mesa/visualization/templates/js/TextModule.js

```diff
@@ -1,7 +1,9 @@
 var TextModule = function() {
-	var tag = "<p></p>";
+	var tag = "<p class='lead'></p>";
```

Comment:
The class usage here doesn't seem to serve any purpose here., either from a CSS rule or used in the frontend code. Is there a use that you have found for it?

Reply:
This is a bootstrap class to "Make a paragraph stand out".
https://getbootstrap.com/docs/4.0/content/typography/#lead

Reply:
Ah okay, cool!